<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>PUBLICATIONS | Yuan  Wang</title>
    <meta name="author" content="Yuan  Wang">
    <meta name="description" content="&lt;p&gt;* indicates co-first authorship.&lt;/p&gt;">
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light">

    <!-- Styles -->
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="http://localhost:4000/publications/">
    
    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark">

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="http://localhost:4000/"><span class="font-weight-bold">Yuan</span>   Wang</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">ABOUT</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/news/">NEWS</a>
              </li>
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">PUBLICATIONS<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/teaching/">TEACHING</a>
              </li>

              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">PUBLICATIONS</h1>
            <p class="post-description"></p>
<p>* indicates co-first authorship.</p>
          </header>

          <article>
            <!-- _pages/publications.md -->
<div class="publications">
  <h2 class="year">2023</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="Chen2023" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Mapping disease regulatory circuits at cell-type resolution from single-cell multiomics data</div>
          <!-- Author -->
          <div class="author">Chen, X, 
                  <em>Wang, Y</em>, Cappuccio, A, Cheng, WS, Zamojski, FR, Nair, VD, Miller, CM, Rubenstein, AB, Nudelman, G, Tadych, A, Theesfeld, CL, Vornholt, A, George, MC, Ruffin, F, Dagher, M, Chawla, DG, Soares-Schanoski, A, Spurbeck, RR, Ndhlovu, LC, Sebra, R, Kleinstein, SH, Letizia, AG, Ramos, I, Jr, VG Fowler, Woods, CW, Zaslavsky, E, Troyanskaya, OG, and Sealfon, SC
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Nature Computational Science</em> Jul 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://doi.org/10.1038/s43588-023-00476-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://github.com/xichensf/magical" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Resolving chromatin-remodeling-linked gene expression changes at cell-type resolution is important for understanding disease states. Here we describe MAGICAL (Multiome Accessibility Gene Integration Calling and Looping), a hierarchical Bayesian approach that leverages paired single-cell RNA sequencing and single-cell transposase-accessible chromatin sequencing from different conditions to map disease-associated transcription factors, chromatin sites, and genes as regulatory circuits. By simultaneously modeling signal variation across cells and conditions in both omics data types, MAGICAL achieved high accuracy on circuit inference. We applied MAGICAL to study Staphylococcus aureus sepsis from peripheral blood mononuclear single-cell data that we generated from subjects with bloodstream infection and uninfected controls. MAGICAL identified sepsis-associated regulatory circuits predominantly in CD14 monocytes, known to be activated by bacterial sepsis. We addressed the challenging problem of distinguishing host regulatory circuit responses to methicillin-resistant and methicillin-susceptible S. aureus infections. Although differential expression analysis failed to show predictive value, MAGICAL identified epigenetic circuit biomarkers that distinguished methicillin-resistant from methicillin-susceptible S. aureus infections.</p>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2021</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="10.1172/JCI148036" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Attenuated activation of pulmonary immune cells in mRNA-1273–vaccinated hamsters after SARS-CoV-2 infection</div>
          <!-- Author -->
          <div class="author">Meyer, M*, 
                  <em>Wang, Y*</em>, Edwards, D*, Smith, GR*, Rubenstein, AB*, Ramanathan, P, Mire, CE, Pietzsch, C, Chen, X, Ge, Y, Cheng, WS, Henry, C, Woods, A, Ma, L, Stewart-Jones, GBE, Bock, KW, Minai, M, Nagata, BM, Periasamy, S, Shi, PY, Graham, BS, Moore, IN, Ramos, I, Troyanskaya, OG, Zaslavsky, E, Carfi, A, Sealfon, SC, and Bukreyev, A
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>The Journal of Clinical Investigation</em> Oct 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://doi.org/10.1172/JCI148036" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The mRNA-1273 vaccine is effective against SARS-CoV-2 and was granted emergency use authorization by the FDA. Clinical studies, however, cannot provide the controlled response to infection and complex immunological insight that are only possible with preclinical studies. Hamsters are the only model that reliably exhibits severe SARS-CoV-2 disease similar to that in hospitalized patients, making them pertinent for vaccine evaluation. We demonstrate that prime or prime-boost administration of mRNA-1273 in hamsters elicited robust neutralizing antibodies, ameliorated weight loss, suppressed SARS-CoV-2 replication in the airways, and better protected against disease at the highest prime-boost dose. Unlike in mice and nonhuman primates, low-level virus replication in mRNA-1273–vaccinated hamsters coincided with an anamnestic response. Single-cell RNA sequencing of lung tissue permitted high-resolution analysis that is not possible in vaccinated humans. mRNA-1273 prevented inflammatory cell infiltration and the reduction of lymphocyte proportions, but enabled antiviral responses conducive to lung homeostasis. Surprisingly, infection triggered transcriptome programs in some types of immune cells from vaccinated hamsters that were shared, albeit attenuated, with mock-vaccinated hamsters. Our results support the use of mRNA-1273 in a 2-dose schedule and provide insight into the potential responses within the lungs of vaccinated humans who are exposed to SARS-CoV-2.</p>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2020</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="menon2020sars" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">SARS-CoV-2 receptor networks in diabetic and COVID-19–associated kidney disease</div>
          <!-- Author -->
          <div class="author">Menon, R*, Otto, EA*, Sealfon, R*, Nair, V*, Wong, AK*, Theesfeld, CL*, Chen, X, 
                  <em>Wang, Y</em>, Boppana, AS, Luo, J, Yang, Y, Kasson, PM, Schaub, JA, Berthier, CC, Eddy, S, Lienczewski, CC, Godfrey, B, Dagenais, SL, Sohaney, R, Hartman, J, Fermin, D, Subramanian, L, Looker, HC, Harder, JL, Mariani, LH, Hodgin, JB, Sexton, JZ, Wobus, CE, Naik, AS, Nelson, RG, Troyanskaya, OG, and Kretzler, M
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Kidney international</em> Oct 2020
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://doi.org/10.1016/j.kint.2020.09.015" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
            <a href="https://hb.flatironinstitute.org/covid-kidney" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>COVID-19 morbidity and mortality are increased via unknown mechanisms in patients with diabetes and kidney disease. SARS-CoV-2 uses angiotensin-converting enzyme 2 (ACE2) for entry into host cells. Because ACE2 is a susceptibility factor for infection, we investigated how diabetic kidney disease and medications alter ACE2 receptor expression in kidneys. Single cell RNA profiling of kidney biopsies from healthy living donors and patients with diabetic kidney disease revealed ACE2 expression primarily in proximal tubular epithelial cells. This cell-specific localization was confirmed by in situ hybridization. ACE2 expression levels were unaltered by exposures to renin-angiotensin-aldosterone system inhibitors in diabetic kidney disease. Bayesian integrative analysis of a large compendium of public -omics datasets identified molecular network modules induced in ACE2-expressing proximal tubular epithelial cells in diabetic kidney disease (searchable at hb.flatironinstitute.org/covid-kidney) that were linked to viral entry, immune activation, endomembrane reorganization, and RNA processing. The diabetic kidney disease ACE2-positive proximal tubular epithelial cell module overlapped with expression patterns seen in SARS-CoV-2–infected cells. Similar cellular programs were seen in ACE2-positive proximal tubular epithelial cells obtained from urine samples of 13 hospitalized patients with COVID-19, suggesting a consistent ACE2-coregulated proximal tubular epithelial cell expression program that may interact with the SARS-CoV-2 infection processes. Thus SARS-CoV-2 receptor networks can seed further research into risk stratification and therapeutic strategies for COVID-19–related kidney damage.</p>
          </div>
        </div>
      </div>
</li></ol>


</div>

          </article>

        </div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2023 Yuan  Wang. <a href="https://www.princeton.edu//" target="_blank" rel="external nofollow noopener">Princeton University</a><a></a>.
Last updated: August 01, 2023.
      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script async src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script async src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script async src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script defer src="/assets/js/common.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
  </body>
</html>
